recipes/cached_property/LICENSE

```diff
@@ -0,0 +1,12 @@
+Copyright (c) 2015, Daniel Greenfeld
```

Comment:
It appears as if you don't need this file since the license is included in the source.